packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -380,3 +393,28 @@ Then('I click the submit button', () => {
     name: new RegExp(`^((submit)|(send code))$`, 'i'),
   }).click();
 });
+
+And('I confirm {string} error is accessible', () => {
```

Comment:
I moved this to `shared.ts`, so that `sign-up-with-errors` and `sign-in-force-new-password` can reuse it.